eZ Platform/APIs/REST/REST_v3.md

```diff
@@ -17,35 +17,42 @@ As a front-end developer I would like to:
 * Manage the content via the REST API:
     * Use the REST endpoints to quickly create a new object
     * Use the REST endpoints to quickly update an already created object
-    * Use the REST endpoint
 
 # Creating content POST
 The create content endpoint should be simplified to default most values and the simplest possible.
 
 This call should create:
 * A new content object (not a draft) and publish
 * Assign ownership to the current user
-* Create the object in the default language
+* Create the object in the default language if not set
 * Location can be defined either with ID or URL
 
 ## Example: creating a new object
 
-```
-Resource: /content/object
-Method: POST
+__Resource__: _/content/object_
+
+__Accept__: _application/vnd.ez.api.Content+json_
+
+__Content-Type__: _application/vnd.ez.api.SimpleContentCreate+json_
+
+__Method__: _POST_
+
+__Body__:
+```json
 {
-    "content-location": "/blogs",
-	"content-type": "blog_post",
+    "ParentLocation": <ID> or <URL>,
```

Comment:
@bdunogier I agree, I just had two concerns when proposing this:
1. ParentLocation taking both path and id might be confusing, i. if we swap to string based id's (UUID) and ii. user might try to use `"/content/locations/1/4/89"` which won't work here
2. More personal preferance but I prefer self describing arguments, and with points in 1. `ParentLocation` is not on it''s own.

But I agree we should make it as simple as possible, so maybe there are other ways we could declare this.